react-backend/client/src/components/visitors/signup_form.js

```diff
@@ -12,21 +13,20 @@ export default props => (
             {props.error.map(el => <span key={el}>{el}</span>)}
           </div>
         )}
-        <Input question="Name" option="fullname" />
-        <Input question="Email" option="email" />
+        <Input question="Name" option="fullname" required />
```

Comment:
AFAICS, this won't have any impact on the actual `input` element inside the custom `Input` component, because additional props (in this case `required`) are not passed through to the child `input` element.

Reply:
What would be the right way of doing this? Would I need to edit the actual input.js component?

Reply:
```js
/* SignupForm.js */
// ...
<Input question="Name" option="fullname" required />
// ...


/* Input.js */
// ...
render () {
  // ...
    <input id={label} {...this.props} />
  // ...
}
// ...
```

Reply:
Lookup "spreading props"